lib/Counter/styles.scss

```diff
@@ -0,0 +1,24 @@
+.counter {
+  font-size: 0.625rem;
+  line-height: 0.5625rem;
+  min-width: 0.875rem;
+}
+
+.button__icon {
```

Comment:
🚩 I feel slightly resistant to this, as this component is now tightly coupled with `<ButtonIcon />`, which I feel like will make maintenance and updating more complex in the long run. I think in the perfect decoupled world, this counter component should not know anything about `<ButtonIcon />`.

I think we could make things less complex if we made the styles component agnostic.

We could add a `border={true/false}` prop to `<Counter />`, which would render the border and apply the border active styles.

I don't feel like the positioning styles should live here ` absolute top-0 right-0 mt-px mr-px`. Maybe we create a new component `ButtonIconCounter` that extends this and applies the specific positioning formatting?

Reply:
I'm going to reference this PR in our frontend tailwind discussion card. I feel like it'll be really relevant there.